packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -297,7 +313,11 @@ class MenuContent extends React.Component {
           onKeyDown,
           isActive,
         });
-      // If the child has children then it is an item group, so iterate through it's children
+
+        if (hasHeaderTitle && item.props.subMenuItems && item.props.subMenuItems.length > 0) {
+          hasSubMenuItems = true;
```

Comment:
I don't think the `hasSubMenuItems` bool is needed, the `isSubMenu` bool below will render sub menu titles